mccarty-dylan/range-of-integers.js

```diff
@@ -0,0 +1,14 @@
+//3 numbers to determine the range of integers
+//input start number
+//input end number 
+//input number the integers increase by 
+//count up to max num until reached max
+
+generateRange(min, max, step); {
+    let arr = [];
+    
+    for(let i = min; i <= max; i += step);
```

Comment:
Missing your {} for the block after the for loop. No semicolon on the end of this line.